### DISCUSSION TITLE
this only lists channels, and not the grid content?

### DISCUSSION THREAD

**Discussion by dwyazzo90** [iptv-org/community#543] [Q&A] [answered]
i thought it showed the live program information and the upcoming shows airing in the next hours on channels but i didnt see anything
sorry if i may sound stupid, i was reviving an Wii U app (Nintendo TVii) and i wanted to implement the program guide but i didnt find anywhere as for an API of listings of programs to air, i hope i get help!

**Comment by BellezaEmporium** [comment]
We used to provide EPGs, but since GH asked us not to use Actions for this kind of content, we've decided to stop providing EPGs. We however are still providing configurations to generate your own EPG schedule.

**Comment by dwyazzo90** [comment]
how? is that info distribuited or not? im really new in this kind of apis stuff

**Reply by BellezaEmporium** [reply, marked as answer]
Everything you need to know is in https://github.com/iptv-org/epg.
